index.html

```diff
@@ -656,6 +664,10 @@ <h4>Whitespace Normalization</h4>
 	    
 	    <section id="accents">
 			<h4>Accents and diacritic marks</h4>
+
+			<p class="advisement">By default, string searching MAY ignore diacritics.</p>
```

Comment:
I know Mozilla provides an option to turn diacritics on. I'm... reluctant to say it this way, though. In general, if I go to the effort to provide a search term with a diacritic in it, I want a more exact match. If I typed `côté` I don't really want to find `cote` or `coté` most of the time (in most cases, though, the user typed `cote` and wants to find all of them by default)

```suggestion
            <p class="advisement" id="need-ids-for-bps">Diacritics in the corpus MAY be ignored when the search term does not contain any.</p>
            <p class="advisement" id="need-another-id">String search SHOULD ignore diacritics by default, unless enabled by the user.</p>
```

Reply:
fixed